test/src/e2e_vm_tests/test_programs/should_pass/language/const_inits/src/main.sw

```diff
@@ -33,12 +32,15 @@ const EN1a = En1::Int(101);
 const EN1b = En1::Arr(ARR2);
 const EN1c = En1::NoVal;
 
-const ETH_ID0_VALUE = ETH_ID0.value;
+
 const TUP1_idx2 = TUP1.2;
 
 fn main() -> u64 {
+    let ETH_ID0 = ~ContractId::from(ZERO_B256);
+    let ETH_ID0_VALUE = ETH_ID0.value;
+
     // initialization through function applications.
-    let eth_id0 = ~ContractId::from(0x0000000000000000000000000000000000000000000000000000000000000000);
+    let eth_id0 = ~ContractId::from(ZERO_B256);
```

Comment:
Ditto. Semantically, `eth_id0` and `eth_id1` use the values zero and one, but there's nothing special about zero specifically so I would avoid using `ZERO_B256`.